tests/unit/experiments/experiment_tests.py

```diff
@@ -9,7 +9,7 @@
 from datetime import timedelta
 
 from baseplate.core import ServerSpan, User, AuthenticationToken
-from baseplate.events import EventQueue
+from baseplate.events import EventQueue, EventLogger
```

Comment:
I don't think you need `EventQueue` anymore.

Reply:
fixed